conf/zookeeper.conf

```diff
@@ -37,10 +37,6 @@ dataLogDir=data/zookeeper/txlog
 # the port at which the clients will connect
 clientPort=2181
 
-# the port at which the admin will listen
-adminPort=9990
-zookeeper.admin.enableServer=true
```

Comment:
pushed button too early....

why do you need to not start ZK admin server ?

Reply:
I believe in zookeeper 3.5.x, the admin server is enabled [by default](https://zookeeper.apache.org/doc/r3.5.4-beta/zookeeperAdmin.html#sc_adminserver).

```
The AdminServer is enabled by default, but can be disabled by either:

Setting the zookeeper.admin.enableServer system property to false.

Removing Jetty from the classpath. (This option is useful if you would like to override ZooKeeper's jetty dependency.)
```

more importantly the settings here are wrong: https://zookeeper.apache.org/doc/r3.5.4-beta/zookeeperAdmin.html#sc_adminserver_config

they should be:

- admin.enableServer 
- admin.serverPort

because these settings are not able to be overwritten based on current docker approach (due to these settings contain '.'), instead of replacing them with right settings, I just removed them to make things simpler.

Reply:
okay